src/Humanizer/Localisation/TimeUnit.cs

```diff
@@ -13,7 +13,8 @@ public enum TimeUnit
         Day,
         Week,
         Month,
-        Year
+        Year,
+        Never
```

Comment:
I don't know whether extending the `TimeUnit` is a good idea. For me `Never` doesn't fit into this concept.
`TimeUnit` values are used in `TimeSpan` humanization as `minTimeUnit` and `maxTimeUnit`, how would this functionality work when `Never` gets passed into?